### DISCUSSION TITLE
invalid datasource when trying to get postgis layer in qfield cloud

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3115] [Q&A]
Hi, 
I am having this error. Please provide me a solution
INFO:root:QGIS project layer checks
Layer Name     Layer Id    Is Valid    Status                Error Summary    Provider Summary
-------------  ----------  ----------  --------------------  ---------------  ------------------
KEast_roads    ...aea30c   True        no_error
P73_boundary   ...4ea8df   True        no_error
P73_buildings  ...4cefe8   False       invalid_dataprovider
Ward_KEast     ...bb5259   True        no_error
Generate Thumbnail Image
INFO:root:Generate project thumbnail image...
WARNING:QGIS_MSGLOG:Connection to database failed
fe_sendauth: no password supplied
WARNING:QGIS_MSGLOG:1 unavailable layer(s) found:
WARNING:QGIS_MSGLOG: * dbname='prabhag122' host= port=5432 sslmode=disable key='uuid' type=MultiPoint checkPrimaryKeyUnicity='1' table="public"."P73_buildings" (geom)
INFO:QGIS_STDERR:Stopping QGIS app...

**Comment by lucienicolier** [comment]
Hi,
It seems QFieldCloud can not connect to your database.
Here you have an explanation how to do it  [Postgis](https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#postgis)

1. Is your database visible from internet
2. Did you set up the [secrets](https://docs.qfield.org/reference/qfieldcloud/secrets/)

**Comment by MonikaShah** [comment]
Hello,
I tried adding the configs to secrets. But still I coudnt access it from
qfield on my mobile.
error that it has is
WARNING:QGIS_MSGLOG:Connection to database failed
fe_sendauth: no password supplied
WARNING:QGIS_MSGLOG:1 unavailable layer(s) found:
WARNING:QGIS_MSGLOG: * dbname='prabhag122' host=162.55.169.58 port=5432
sslmode=disable key='uuid' type=MultiPoint checkPrimaryKeyUnicity='1'
table="public"."P73_buildings" (geom)
INFO:QGIS_STDERR:Stopping QGIS app.

On Tue, Jul 26, 2022 at 2:57 PM Lucie Nicolier ***@***.***>
wrote:

> Hi,
> It seems QFieldCloud can not connect to your database.
> Here you have an explanation how to do it Postgis
> <https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/#postgis>
>
>    1. Is your database visible from internet
>    2. Did you set up the secrets
>    <https://docs.qfield.org/reference/qfieldcloud/secrets/>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QField/discussions/3115#discussioncomment-3229481>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA76P5PEP4A54NOTPMFBIY3VV6VPVANCNFSM54U4SYUQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
With Regards,
Monika Shah

**Comment by suricactus** [comment]
@MonikaShah seems you have configured you PostGIS layer improperly. Are you sure you configured your layer to require a service with specific name?

**Reply by australgisinc** [reply]
How you can do this. Can you be more specific about "configuring a layer properly"?

**Comment by MonikaShah** [comment]
Thanks for your response after almost 4 years.

On Tue, Jun 2, 2026 at 12:57 AM CamiloPA ***@***.***> wrote:

> How you can do this. Can you be more specific about "configuring a layer
> properly"?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QField/discussions/3115?email_source=notifications&email_token=AA76P5KG4O2S6ERFB4HBR5345XKJ5A5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZRGQYTCNBQUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17141140>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA76P5KIJXM3OE73SQV4MI345XKJ5AVCNFSM6AAAAACZWBGM7CVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTOMJUGEYTIMA>
> .
> Triage notifications, keep track of coding agent tasks and review pull
> requests on the go with GitHub Mobile for iOS
> <https://github.com/notifications/mobile/ios/AA76P5NFPVZ3HQVESUXISZT45XKJ5A5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZRGQYTCNBQUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ>
> and Android
> <https://github.com/notifications/mobile/android/AA76P5LPC4VSYO3YP3FDGJL45XKJ5A5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZRGQYTCNBQUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>.
> Download it today!
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
With Regards,
Monika Shah
